app/models/target.rb

```diff
@@ -34,8 +34,6 @@ def get_progress
         conference.registrations.where('created_at < ?', due_date).count
       when Target.units[:program_minutes]
         conference.current_program_minutes
-      else
-        0
```

Comment:
Do you guys want me to add it back? I would still say that
> In case something goes wrong with our validations, we won't end up with nil value

is not something we should worry about. Cause if we do, we will end up with `if-else` blocks all over the code case. For ex (from user model):
```Ruby
validates :email, presence: true

validates :username, presence: true

def self.find_for_auth(auth, current_user = nil)
  # omitted code//

  if user.new_record?
    if auth.info.email
      user.email = auth.info.email
    else
      # do something else
    end
    user.name = auth.info.name
    if auth.info.username
      user.username = auth.info.username
    else
      # do something else
    end
    user.password = Devise.friendly_token[0, 20]
    user.skip_confirmation!
  end

  user
end
```